content/recipes/nestia.md

```diff
@@ -0,0 +1,698 @@
+
+[![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/samchon/nestia/blob/master/LICENSE)&nbsp;[![npm version](https://badge.fury.io/js/nestia.svg)](https://www.npmjs.com/package/nestia)&nbsp;[![Downloads](https://img.shields.io/npm/dm/nestia.svg)](https://www.npmjs.com/package/nestia) &nbsp;[![Build Status](https://github.com/samchon/nestia/workflows/build/badge.svg)](https://github.com/samchon/nestia/actions?query=workflow%3Abuild)&nbsp;[![Guide Documents](https://img.shields.io/badge/wiki-documentation-forestgreen)](https://github.com/samchon/nestia/wiki) 
+
+  - Github: https://github.com/samchon/nestia
+  - NPM: https://www.npmjs.com/packages/nestia
+  - Guide Documents: https://github.com/samchon/nestia/wiki
+
+```bash
+# INSTALL NESTIA
+npm install --save-dev nestia
+
+# BUILDING SDK LIBRARY
+npx nestia sdk "src/controller" --out "src/api"
+npx nestia sdk "src/**/*.controller.ts" --out "src/api"
+npx nestia sdk "src/controller" \ 
+    --exclude "src/controller/test" \
+    --out "src/api"
+
+# BUILDING SWAGGER.JSON IS ALSO POSSIBLE
+npx nestia swagger "src/controller" -- out "swagger.json"
+```
+
+Don't write any swagger comment and DTO decorator. Just run the `nestia` up.
+
+  - No swagger comment/decorator
+  - No DTO comment/decorator
+  - Only pure `interface`s and NestJS code are required
+    - Support generic typed `interface`s and `controller`s
+    - Support union/intersection types
+    - Support conditional types
+
+The `nestia` is an evolved automatic `SDK` and `Swagger` generator than ever, who does not require any type of swagger comment or DTO decorator function, by analyzing your `NestJS` developed backend server code in the compilation level (`nestia` utilizes the TypeScript Compiler API).
+
+Therefore, don't write any swagger comment and don't use any DTO related decorator function. Just use the pure interface type with the pure `NestJS` code. Reading below sections and looking at example codes of the `nestia`, feel how the `nestia` is much stronger than the legacy `@nestjs/swagger`.
+
+<img src="https://github.com/samchon/nestia/wiki/images/sdk-comparison-table.png" style="max-width: 100%" />
+
+```typescript
+// IMPORT SDK LIBRARY WHO'VE BEEN GENERATED BY THE NESTIA
+import api from "@samchon/shopping-api";
+import { IPage } from "@samchon/shopping-api/lib/structures/IPage";
+import { ISale } from "@samchon/shopping-api/lib/structures/ISale";
+import { ISaleArticleComment } from "@samchon/shopping-api/lib/structures/ISaleArticleComment";
+import { ISaleQuestion } from "@samchon/shopping-api/lib/structures/ISaleQuestion";
+
+export async function trace_sale_question_and_comment
+    (connection: api.IConnection): Promise<void>
+{
+    // LIST UP SALE SUMMARIES
+    const index: IPage<ISale.ISummary> = await api.functional.shoppings.sales.index
+    (
+        connection,
+        "general",
+        { limit: 100, page: 1 }
+    );
```

Comment:
Is this how prettier is generating the TS to be formatted? I could've sworn we had `(` on the same line as the function instead of the next line